addons/website_crm_score/controllers/main.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+from openerp import addons, http, SUPERUSER_ID, fields
+from openerp.http import request
+
+
+class PageController(addons.website.controllers.main.Website):
+
+    @http.route('/page/<page:page>', auth="public", website=True)
+    def page(self, page, **opt):
+        response = super(PageController, self).page(page, **opt)
+        # duplication of ir_http.py
+
+        if hasattr(response, 'status_code') and response.status_code == 200:
```

Comment:
Response objects always have status codes. Even if that wasn't the case, `getattr` would be more suitable.